vnext/Scripts/copyRNLibraries.js

```diff
@@ -14,16 +14,25 @@ const rnCopiesDir = path.join(
   path.dirname(require.resolve('react-native-windows/package.json')),
   'ReactCopies',
 );
+const rnTypesDir = path.dirname(
+  require.resolve('@types/react-native/package.json'),
+);
 
 exports.copyTask = baseDir => {
   const reactNative = (...files) => files.map(f => path.join(rnDir, f));
+  const reactNativeTypes = (...files) =>
+    files.map(f => path.join(rnTypesDir, f));
   const reactCopies = (...files) => files.map(f => path.join(rnCopiesDir, f));
   const src = (...files) => files.map(f => path.join(baseDir, 'src', f));
   const base = file => path.join(baseDir, file);
 
   return series(
     exports.cleanTask(baseDir),
 
+    copyTask({
```

Comment:
Does copyTask support `robocopy /mir`? Style copy or should this folder be added to the 'clean' target as well?
Otherwise removed stale files will never be cleared picked up without a `git clean -xdf`